### ISSUE TITLE
Updates to values.yaml not reflected in release (e.g. Flux not updating)

### ISSUE THREAD

**Issue by boocup** [fluxcd/helm-operator#272]
**Describe the bug**

App's chart being correctly deployed by Flux, but updates to values.yaml not being in configmap.

**To Reproduce**

Deploy App's Helm chart into cluster
Check values by using 'kubectl describe cm tidal-config'
Modify values.yaml and update the git repository
Monitor log file, watch for sync-loop
Check values again by using 'kubectl describe cm tidal-config'
Bash into the flux pod and confirm git sync did pull down new values.yaml (which is confirmed)
Confirm mappings in template file are correct (which is confirmed)

**Expected behavior**

Would expect 'kubectl describe cm tidal-config' to reflect updates

**Log**

Snippet of log (note unchanged)

ts=2020-02-05T20:50:36.5183184Z caller=sync.go:482 method=Sync cmd=apply args= count=2
ts=2020-02-05T20:50:36.7073536Z caller=sync.go:548 method=Sync cmd="kubectl apply -f -" took=188.9599ms err=null output="storageclass.storage.k8s.io/local-storage unchanged\nhelmrelease.helm.fluxcd.io/tidal unchanged"
ts=2020-02-05T20:52:52.204638Z caller=warming.go:198 component=warmer info="refreshing image" image=docker.io/fluxcd/helm-operator tag_count=29 to_update=29 of_which_refresh=29 of_which_missing=0
ts=2020-02-05T20:52:52.9987441Z caller=warming.go:206 component=warmer updated=docker.io/fluxcd/helm-operator successful=29 attempted=29
ts=2020-02-05T20:53:58.0244469Z caller=images.go:17 component=sync-loop msg="polling for new images for automated workloads"

**Additional context**

- Helm operator version: 2.13
- Kubernetes version: 1.15
- Git provider: GitHub.com

**Comment by boocup** [comment]
So, it appears that after my app's chart is deployed, if I do a first subsequent values.yaml, commit it, push it and wait for the sync.  I see the updated value in the config map.

When I do another update to the values.yaml after that first one, I don't see an updated configmap value via kubectl describe.

But, if I bash into the flux pod and look at the most recent flux-working folder, I see the update on the sync'd values.yaml file. 

Should flux be updating the helm chart with this second update (and not just the first update after the initial deployment).

To summarize.

1) initial helm chart launch by flux - OK
2) update values.yaml, push and wait for sync to see change - OK
3) another update to values.yaml, push and wait for sync to see change - kubectl describe doesn't show update, but update visible in most recent flux-working dir in flux pod.

Any suggestions? Maybe I don't have the configuration correct.

**Comment by stefanprodan** [comment]
Flux plays no role in applying changes from charts stored in git. It's Helm Operator that clones the repo where the chart is. Helm Operator merges values.yaml with the HelmRelease.values and if the dry-run detects a change, only then the release will be upgraded. Can you please check helm-op logs?

**Comment by boocup** [comment]
will do....thanks @stefanprodan !

**Comment by boocup** [comment]
good tip....I'm going to do some refactoring. I'll close this. Thanks again.

**Comment by hiddeco** [comment]
@boocup in the end you did not close the issue, does this mean it is still an issue or did you forget to close it after commenting?

**Comment by boocup** [comment]
my apologies, I thought I did close it....closing now.
